src/Sulu/Bundle/MediaBundle/Resources/js/views/MediaOverview/MediaOverview.js

```diff
@@ -144,8 +144,33 @@ class MediaOverview extends React.Component<ViewProps> {
         this.collectionId.set(collectionId);
     };
 
-    @action handleUploadError= () => {
-        this.errors.push(translate('sulu_media.upload_server_error'));
+    @action handleUploadError= (errors: Array<Object>) => {
+        if (errors.length !== 1) {
+            this.errors.push(translate('sulu_media.upload_server_error'));
+
+            return;
+        }
+
+        switch (errors[0].code) {
```

Comment:
once https://github.com/sulu/sulu/pull/5799#discussion_r571003469 is settled, we should use the same mechanism here